inference/enterprise/device_manager/server.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This is just the CA certificate and doesn't contain the private key. It's used to verify the authenticity of the server certificate during TLS/SSL handshakes (same way a browser would verify a website has https). It doesn't pose a security risk in itself because it can't be used to decrypt traffic or impersonate identities.